docs/getting_started.md

```diff
@@ -0,0 +1,110 @@
+
```

Comment:
Unnecessary empty lines at the beginning?

Reply:
Resolved, thanks! :D